src/Elmah/XmlFileErrorLog.cs

```diff
@@ -78,6 +79,20 @@ public XmlFileErrorLog(IDictionary config)
                 logPath = MapPath(logPath);
 
             _logPath = logPath;
+
+
+            var fileListSizString = config.Find("size", string.Empty);
+
+            if (fileListSizString.Length == 0)
```

Comment:
If size is not given then there should be no limit for backward compatibility. It certainly should not have any relation to `MemoryErrorLog` defaults and sizes.